README.md

```
@@ -1,6 +1,14 @@
+<img src="https://madewithlove.now.sh/af?text=Happy+Hackking+Space" alt="Made with love in Happy Hackking Space">
+
 # funURL
+
+<img src="img/image.png"> <br>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)